config/routes.rb

```diff
@@ -503,6 +503,7 @@
   get "svg-sprite/:hostname/svg-:theme_ids-:version.js" => "svg_sprite#show", constraints: { hostname: /[\w\.-]+/, version: /\h{40}/, theme_ids: /([0-9]+(,[0-9]+)*)?/, format: :js }
   get "svg-sprite/search/:keyword" => "svg_sprite#search", format: false, constraints: { keyword: /[-a-z0-9\s\%]+/ }
   get "svg-sprite/picker-search" => "svg_sprite#icon_picker_search", defaults: { format: :json }
+  get "svg-sprite/icon(/:color)/:name.svg" => "svg_sprite#svg_icon", constraints: { name: /[-a-z0-9\s\%]+/, format: :svg }
```

Comment:
I think this needs a `:hostname` parameter, since the action in the controller checks for `params[:hostname]`